docs: improve the kubeFleet landing page and add community meeting info (#67)
kubefleet-dev/kubefleet: README.md

KubeFleet is a sandbox project of the [Cloud Native Computing Foundation](https://cncf.io/) (CNCF) that works on any Kubernetes cluster.
We are working towards the vision where we will eventually be able to treat each Kubernetes cluster as a [cattle](https://cloudscaling.com/blog/cloud-computing/the-history-of-pets-vs-cattle/).

## What is KubeFleet?
KubeFleet contains a set of Kubernetes controllers and CRDs which provide an advanced cloud-native solution for multi-cluster application management.

Use KubeFleet to schedule workloads smartly, roll out changes progressively, and perform administrative tasks easily, across a group of Kubernetes clusters on any cloud or on-premises clusters.

## Quickstart

* [Get started here](https://kubefleet-dev.github.io/website/docs/getting-started/kind/)

## Documentation

> To learn more about KubeFleet go to the [the KubeFleet documentation](https://kubefleet-dev.github.io/website/).

Reviewer: ```suggestion
To learn more about KubeFleet go to the [KubeFleet documentation](https://kubefleet-dev.github.io/website/).
```
![image](https://github.com/user-attachments/assets/135337e1-c3e0-4c24-ace3-c8335f50f221)

> To learn more about KubeFleet go to the [the KubeFleet documentation](https://kubefleet-dev.github.io/website/).

Reviewer 2: Hi Ryan! I think the duplicate `the` issue re-appears somehow

Reviewer 2: Same to a lot of the raised issues... Is there something wrong on my end?

Author: fixed, do you see more duplicates?

## Community

You can reach the KubeFleet community and developers via the following channels:

* Q & A: [Github Discussions](https://github.com/kubefleet-dev/kubefleet/discussions)

> * Slack: [The #KubeFleet Slack channel] (https://cloud-native.slack.com/archives/C08KR7589R8) 

Reviewer: ```suggestion
* Slack: [The #KubeFleet Slack channel](https://cloud-native.slack.com/archives/C08KR7589R8) 
```
![image](https://github.com/user-attachments/assets/12529f68-7366-4035-85e9-9f1c857bdf79)

> * Mailing list: - [mailing list](https://groups.google.com/g/kubefleet-dev)

Reviewer: ```suggestion
* Mailing list: [mailing list](https://groups.google.com/g/kubefleet-dev)
```

## Community Meetings

> We host bi-weekly community meetings that alternate between US/EU and APAC friendly time. The community showcase demos and discuss the current and future state of the project in those meetings.

Reviewer: ```suggestion
We host bi-weekly community meetings that alternate between US/EU and APAC friendly time. In these sessions the community will showcase demos and discuss the current and future state of the project.
```

Please refer to the [KubeFleet CNCF meeting calendar](https://zoom-lfx.platform.linuxfoundation.org/meetings/kubefleet?view=month) for the latest schedule.

> For more meeting information, minutes and recordings, please see the [KubeFleet community meeting doc](https://docs.google.com/document/d/1iMcHn11fPlb9ZGoMHiGEBvdIc44W1CjZvsPH3eBg6pA/edit?usp=sharing)

Reviewer: ```suggestion
For more meeting information, minutes and recordings, please see the [KubeFleet community meeting doc](https://docs.google.com/document/d/1iMcHn11fPlb9ZGoMHiGEBvdIc44W1CjZvsPH3eBg6pA/edit?usp=sharing).
```

## Support

> For more information, see [SUPPORT.md](SUPPORT.md).

Reviewer: ```Suggestion
For more information, see [SUPPORT](SUPPORT.md).
```

[1]:  https://img.shields.io/github/v/release/kubefleet-dev/kubefleet
[2]:  https://goreportcard.com/badge/go.goms.io/fleet
[3]:  https://goreportcard.com/report/go.goms.io/fleet

> [4]:  https://codecov.io/gh/Azure/fleet/branch/main/graph/badge.svg?token=D3mtbzACjC

Reviewer 2: Oh, right. We have a few more badges to add; let me create a work item
